### PR TITLE
Formatting everything

### PR DESCRIPTION
**Pull request by abejgonzalez**
Use clang-format, scalafmt, scalafix to fix/format all the C/C++/Scala code in Chipyard

TODO:
- [ ] Add CI
- [ ] Add docs on how to run things (also add it in the CI)

**Related PRs / Issues**:

**Type of change**:
- [ ] Bug fix
- [ ] New feature
- [ ] Other enhancement

**Impact**:
- [ ] RTL change
- [ ] Software change (RISC-V software)
- [ ] Build system change
- [ ] Other

**Contributor Checklist**:
- [ ] Did you set `main` as the base branch?
- [ ] Is this PR's title suitable for inclusion in the changelog and have you added a `changelog:<topic>` label?
- [ ] Did you state the type-of-change/impact?
- [ ] Did you delete any extraneous prints/debugging code?
- [ ] Did you mark the PR with a `changelog:` label?
- [ ] (If applicable) Did you update the conda `.conda-lock.yml` file if you updated the conda requirements file?
- [ ] (If applicable) Did you add documentation for the feature?
- [ ] (If applicable) Did you add a test demonstrating the PR?

- [ ] (If applicable) Did you mark the PR as `Please Backport`?

### CODE REVIEW

File: fpga/src/main/resources/vcu118/sdboot/include/bits.h [hunk 1]
```diff
@@ -5,31 +5,32 @@
...
 #define XSTR(x) #x
 
 #if __riscv_xlen == 64
-# define SLL32    sllw
-# define STORE    sd
-# define LOAD     ld
-# define LWU      lwu
-# define LOG_REGBYTES 3
+#define SLL32 sllw
+#define STORE sd
```
Comment (jerryz123) [inline fpga/src/main/resources/vcu118/sdboot/include/bits.h:24]: Bad change

File: fpga/src/main/resources/vcu118/sdboot/include/const.h [hunk 2]
```diff
@@ -5,14 +5,14 @@
...
 #else
-#define _AC(X,Y)        (X##Y)
-#define _AT(T,X)        ((T)(X))
+#define _AC(X, Y) (X##Y)
+#define _AT(T, X) ((T)(X))
 #endif /* !__ASSEMBLER__*/
 
-#define _BITUL(x)       (_AC(1,UL) << (x))
-#define _BITULL(x)      (_AC(1,ULL) << (x))
+#define _BITUL(x) (_AC(1, UL) << (x))
```
Comment (jerryz123) [inline fpga/src/main/resources/vcu118/sdboot/include/const.h:15]: Bad change

File: fpga/src/main/resources/vcu118/sdboot/include/devices/gpio.h [hunk 3]
```diff
@@ -3,22 +3,22 @@
...
-#define GPIO_HIGH_IP    (0x2C)
-#define GPIO_LOW_IE     (0x30)
-#define GPIO_LOW_IP     (0x34)
-#define GPIO_IOF_EN     (0x38)
-#define GPIO_IOF_SEL    (0x3C)
-#define GPIO_OUTPUT_XOR    (0x40)
+#define GPIO_PULLUP_EN (0x10)
+#define GPIO_DRIVE (0x14)
+#define GPIO_RISE_IE (0x18)
+#define GPIO_RISE_IP (0x1C)
```
Comment (jerryz123) [inline fpga/src/main/resources/vcu118/sdboot/include/devices/gpio.h:13]: Bad change

Comment (abejgonzalez) [reply fpga/src/main/resources/vcu118/sdboot/include/devices/gpio.h]: By "bad change" I assume you meant we should align macros. I've added that in.

**Review by jerryz123 — CHANGES_REQUESTED** [review]
Rejecting on the grounds of:
- This requires quite a lot of effort to review
- This can be done on a case-by-case basis when people read the code
- This induces merge conflicts for those using forks of chipyard, who need to apply patches from upstream.

**Comment by abejgonzalez** [comment]
And it's gone!

**Pull request closed without merging**